examples/static_graph_optimizations/mnist/README.md

```diff
@@ -0,0 +1,7 @@
+# Multi-Layer Perceptron for MNIST Classification
```

Comment:
Add `with static subgraph optimization` to make clear how this is different from the other one.